react-basic-practice/src/components/TableColumn/index.tsx

```diff
@@ -71,10 +71,13 @@ const TableColumn: ICustomerColumn[] = [
   },
   {
     key: "options",
-    value: (data: ICustomer) => (
+    value: (
+      data: ICustomer,
+      action?: (type: "Add" | "Edit" | "View", id?: number) => void
```

Comment:
Pls create type for `"Add" | "Edit" | "View"` and reuse it for all.

Reply:
i done

Reply:
i done